pynicotine/transfers.py

```diff
@@ -525,10 +522,14 @@ def update_ui(self, stat_id):
 
         events.emit("update-stat-value", stat_id, session_stat_value, total_stat_value)
 
+    def refresh_stats(self):
```

Comment:
It would be better if we don't mention "ui". Might as well rename this method to `update_stats`, and `update_ui` to `update_stat`.

Reply:
I concur.